### ISSUE TITLE
[Ratman Ralph] Differences with the Flash version

### ISSUE THREAD

**Issue by Toad06** [ruffle-rs/ruffle#815]
Flash: https://www.newgrounds.com/portal/view/67006
Ruffle: https://www.newgrounds.com/portal/view/67006/format/flash?emulate=flash

This game is definitely playable but has some differences with the Flash version.

**Text misalignment**

On the title screen, click "Options", then "Next". The text will be misaligned (because of clickable links?). You can check the other screens by clicking "Next" again.
![r1](https://user-images.githubusercontent.com/14274782/86532902-907e7080-becd-11ea-9a93-a6e8cd21999c.png) ![r1f](https://user-images.githubusercontent.com/14274782/86532905-97a57e80-becd-11ea-9171-e96938ebcda1.png)

**Minor graphic issue**

When the rat eats poison, he loses a life and gets back to his start position. However, his tail remains on the board until you press the up arrow key.
![r2](https://user-images.githubusercontent.com/14274782/86532915-abe97b80-becd-11ea-904e-bee904c138a9.png) ![r2f](https://user-images.githubusercontent.com/14274782/86532918-adb33f00-becd-11ea-9f6b-919cd6dcd779.png)

**Board not cleared after game over**

When you lose all your lives, you get back to the title screen but some objects from the last game remain on screen. Also, if you click the "Options" button, nothing will happen; if you click "Instructions" you will start a new game.
![r3](https://user-images.githubusercontent.com/14274782/86532920-b73ca700-becd-11ea-85a4-7a7518d0ff6c.png) ![r3f](https://user-images.githubusercontent.com/14274782/86532921-b9066a80-becd-11ea-937e-6c9a9107d551.png)

**Comment by Toad06** [comment]
The text stuff has been fixed by #1040. Now just remains the issue with buttons having a different behavior, after a "game over" ("Options" button does nothing will happen and "Instructions" button starts a new game).

**Comment by Lord-McSweeney** [comment]
Still occurring on latest nightly.

**Comment by Toad06** [comment]
Yup, none of the PRs that would have fixed the issue was merged. :(
